tests/gatling_kickoff.rb

```diff
@@ -1,6 +1,26 @@
-step "Execute gatling scenario" do
-  on metric, "cd /root/gatling-puppet-load-test/simulation-runner/ && " +
-      "PUPPET_GATLING_MASTER_BASE_URL=https://#{master.hostname}:8140 " +
-      "PUPPET_GATLING_SIMULATION_CONFIG=config/scenarios/ApplesToApples.json " +
-      "PUPPET_GATLING_SIMULATION_ID=PerfTestLarge sbt run"
+require 'beaker-benchmark'
+
+test_name 'Kickoff Gatling' do
+
+  atop_log = '/tmp/atop.log'
+  start_monitoring(master, atop_log, true)
+
+  begin
+    step "Execute gatling scenario" do
+      on metric, "cd /root/gatling-puppet-load-test/simulation-runner/ && " +
+          "PUPPET_GATLING_MASTER_BASE_URL=https://#{master.hostname}:8140 " +
+          "PUPPET_GATLING_SIMULATION_CONFIG=config/scenarios/ApplesToApples.json " +
+          "PUPPET_GATLING_SIMULATION_ID=PerfTestLarge sbt run",
+         {:accept_all_exit_codes => true}
+    end
+
+  ensure
+    perf_result = stop_monitoring(master, 'Apples to Apples gatling run', atop_log, '/opt/puppetlabs')
+    if perf_result
+      perf_result.log_summary
+      # Write summary results to log so it can be archived
+      perf_result.log_csv "PERF_#{today}/atop_summary.csv"
```

Comment:
maybe you have some requires and such elsewhere that make the "today" work, but standalone this won't run.  Also, if doing manual runs, and you do more than one a day, one will overwrite the other.